pinterest/teletraan
# Groups config page needs subnets and public IP UI updates

In order render the UI components from the Cluster settings, the cluster needs to exist. The Group Launch Configuration page is a "create" page if the Group does not exist, but the create button is greyed out  even if you populate the fields. 

Solution

- render a message to the user to create a cluster in their environment rather than rendering the Group Launch Configuration. Even if they were able to create a group it would be associated with any environment or stage. 

Test Plan:

- Navigate to the following URL path /groups/{{group that does not exist}}/config/
- Confirm that an error message page is rendered
- <img width="868" alt="Screen Shot 2022-08-25 at 2 39 12 PM" src="https://user-images.githubusercontent.com/69278532/186773989-f386fa72-15ce-48e8-b900-f07569873059.png">
- Navigate to the following URL path /groups/{{group that does exist}}/config/
- Confirm that the Group Configuration page renders
- <img width="866" alt="Screen Shot 2022-08-25 at 2 38 58 PM" src="https://user-images.githubusercontent.com/69278532/186773981-bad4805e-53fa-43dc-8d50-48958c7784f1.png">
